usaspending_api/disaster/v2/views/agency/count.py

```diff
@@ -26,12 +26,9 @@ def post(self, request: Request) -> Response:
         ]
 
         if self.award_type_codes:
-            filters.append(self.is_non_zero_award_spending)
-            count = (
-                FinancialAccountsByAwards.objects.filter(*filters)
-                .values("award_id")
-                .aggregate(count=Count("award__funding_agency__toptier_agency", distinct=True))["count"]
-            )
+            count = when_non_zero_award_spending(
+                FinancialAccountsByAwards.objects.filter(*filters).values("award__funding_agency__toptier_agency")
+            ).aggregate(count=Count("award__funding_agency__toptier_agency", distinct=True))["count"]
 
         else:
```

Comment:
Might need to ask a SME about this execution path.

Reply:
I assume the `is_non_zero_total_spending`? I've been trying to tackle one miracle at a time.

Reply:
Those appear to be on the record level, not the aggregation values

Reply:
Added comment to ticket for future tracking